### PR TITLE
F/use chakra components

### PR DESCRIPTION
**Pull request by selankon**
- [x] Organization components
- [x] Tags
- [x] Election components

### CODE REVIEW

File: components/elements/CustomTag.tsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { Tag } from '@chakra-ui/tag';
+import React, { ReactNode } from 'react';
+
+type TagProps = React.ComponentProps<typeof Tag>;
+
+export const CustomTag = ({ children, ...rest }: { children: ReactNode } & TagProps) => (
```
Comment (elboletaire) [inline components/elements/CustomTag.tsx:6]: Recommended to use `PropsWithChildren` for these cases:

```tsx
export const CustomTag = ({ children, ...rest }: PropsWithChildren<TagProps>) => (
```

File: components/pages/organizations/components/OrganizationCard.tsx [hunk 2]
```diff
@@ -82,22 +74,15 @@ export const OrganizationCard = ({
   const link = getOrganizationPath(organizationId);
 
   const id = organization?.address ?? organizationId ?? '';
-  const orgName = organization?.account?.name.default.length === 0 ? id : organization?.account?.name.default;
+  // const orgName = organization?.account?.name.default.length === 0 ? id : organization?.account?.name.default;
```
Comment (elboletaire) [inline components/pages/organizations/components/OrganizationCard.tsx:77]: you may probably wanna remove this

**Review by elboletaire — CHANGES_REQUESTED** [review]

File: components/pages/organizations/components/OrganizationCard.tsx (hunk 2 above)
Comment (selankon) [reply components/pages/organizations/components/OrganizationCard.tsx]: e855708

File: components/elements/CustomTag.tsx (hunk 1 above)
Comment (selankon) [reply components/elements/CustomTag.tsx]: f3b1af4

**Review by elboletaire — APPROVED** [review]

**Pull request merged**